.vib/keycloak/cypress/cypress/integration/keycloak_spec.js

```diff
@@ -0,0 +1,87 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login/logout', () => {
+  cy.login();
+  cy.get('#input-error').should('not.exist');
+  cy.get('.dropdown-toggle').click({ force: true });
+  cy.contains('Sign Out').click();
+  cy.contains('#kc-login', 'Sign In');
+});
+
+it('allows changing user data', () => {
+  cy.login();
+  cy.get('.dropdown-toggle').click({ force: true });
+  cy.contains('a', 'Manage account').click({ force: true });
```

Comment:
I don't know if there is an specific URL to access this, but we should prefer `cy.visit()` over interacting with the UI to reach a page.

Reply:
There is an URL but I would prefer not to use it, it's not a clean route: 

![Screenshot 2022-04-29 at 11 59 20](https://user-images.githubusercontent.com/47592680/165924130-56bdd52e-2efb-4a84-a6b3-f6d652dd3f4c.png)